### PR TITLE
[refactor] Use camelCase in thrift generated java sources

### LINKED ISSUE #4437: [refactor]   Use Camel case for our thrift generated source
At present, due to historical reasons, my code has mixed underscore style and camel case style in thrift code. Thrift has an option `fullcamel` to automatically convert the generated code style when generating Java code. We should enable this option to make us fe's code style is more unified

### PR DESCRIPTION
**Pull request by yangzhg**
## Proposed changes

Use camelCase in thrift generated java sources to make us fe's code style is more unified

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [x] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have create an issue on (Fix #4437), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by EmmyMiao87 — COMMENTED** [review]
The local test should be executed. 
Do you test the upgrade operator?

File: fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java [hunk 1]
```diff
@@ -17,16 +17,16 @@
 
 package org.apache.doris.analysis;
 
-import org.junit.Assert;
```
Comment (kangkaisen) [inline fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java:20]: Please update your IDE code style setting, don't introduce this unnecessary change.

**Review by kangkaisen — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java (hunk 1 above)
Comment (yangzhg) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java:20]: @kangkaisen

Comment (kangkaisen) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java:20]: I mean all xxxTest files, not only this test file.

Comment (yangzhg) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java]: @kangkaisen  all done.  please review it again

**Review by kangkaisen — DISMISSED** [review]
+1

**Review by morningman — APPROVED** [review]
LGTM
